app/settings/surveys/targeted-surveys/targeted-survey-edit.html

```diff
@@ -140,35 +141,53 @@ <h2 class="stepper-heading">
                     <ng-form name="stepThree">
                         <div class="stepper-item" ng-class="{'active': isActiveStep(3), 'disabled': !isActiveStep(3)}">
                             <h2 class="stepper-heading">
-                                    <span class="stepper-badge">3</span>
-                                        Add your questions
+                            <span class="stepper-badge">3</span>
+                            <span translate="survey.targeted_survey.add_questions_title"></span>
                             </h2>
-                                <div ng-show="isActiveStep(3)">
-                                     <div class="form-field">
-                                        <div class="button-group">
-                                            <button class="button-alpha" ng-disabled="!isStepComplete(targetedSurvey.stepThree)" ng-click="completeStepThree()">Continue</button>
-                                            <button class="button" ng-click="previousStep(); resetNumbers()">Back</button>
-                                        </div>
+                            <div class="alert" ng-show="isActiveStep(3)">
+                                <p translate="survey.targeted_survey.add_questions_description"></p>
+                            </div>
+                            <div class="form-field" ng-show="isActiveStep(3)">
+                                <ul id="listWithHandle" class="sortble-list">
+                                    <li class="sortable-list-item" ng-repeat="question in stepThree.questions" data={{question.label}}>
+                                        <span class="list-handle">
+                                            <svg class="iconic">
+                                                 <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/img/iconic-sprite.svg#grid-four-up"></use>
```

Comment:
This will be fixed in the next PL-release
And the Icon is ok, see comment here: https://app.zenhub.com/workspace/o/ushahidi/platform/issues/2557#issuecomment-371958911